contracts/SignHash.sol

```diff
@@ -0,0 +1,32 @@
+pragma solidity ^0.4.0;
+
+
+contract SignHash {
+
+    //--- Storage
+
+    mapping (bytes32 => address[]) private hashSigners;
```

Comment:
I'm wondering, how do you intend to store Signer "identities" to verify (I belive we don't have a word for that, I mean their domain name / github organization)? As instances of another contract?

Reply:
I see it as a `mapping (address => Signer)` with `struct Signer` containing information about registered identity verification methods.

Reply:
👍 I'm thinking we might need to use mappings to store (`verification_source` -> `identity`, eg. `domain`: `signhash.com`) for each signer to make that future proof. But if you have something else in mind that's a discussion for a separate issue.

Reply:
:+1: Yes, this is a good idea, I was thinking about extensibility without redeployment of the contract but its a separate issue.